tests/integration/tests/autocomplete.js

```diff
@@ -306,8 +305,27 @@ test('Retrieve restaurant category when we search "restau"', async() => {
   expect(suggestionId).toEqual('category:restaurant');
 });
 
+test('Retrieve no category when we search "barcelona", not even "bar"', async() => {
+  const searchQuery = 'barcelona';
+
+  responseHandler.addPreparedResponse(mockAutocomplete, /autocomplete\?q=barcelona/);
+
+  await page.goto(APP_URL);
+  await page.keyboard.type(searchQuery);
+  await page.waitForSelector('.autocomplete_suggestion');
+
+  const firstLine = await page.evaluate(() => {
+    return document.querySelector(
+      '.autocomplete_suggestion:first-child .autocomplete_suggestion__first_line'
+    ).innerText;
+  });
+
+  expect(firstLine).toEqual('Test Result 1');
```

Comment:
Strange that it's equal, as the mock data isn't capitalized :thinking:

Reply:
Good point. A css rule had been defined to capitalize suggestions results, but this should be applied on category suggestions only. It's now fixed.